styles.css

```diff
@@ -382,8 +435,372 @@ textarea {
   margin: 1% 2%;
 }
 
+@media (max-width: 768px) {
+  .desktop {
+    display: none;
+  }
+
+  .logo.active {
+    visibility: hidden;
+    transition: 0s;
+  }
+
+  .nav-menu {
+    position: fixed;
+    flex-direction: column;
+    left: -100%;
+    gap: 0;
+    font-size: 32px;
+    width: 100%;
+    height: 100%;
+    text-align: left;
+    mix-blend-mode: normal;
+    transition: 0.5s ease-in-out;
+  }
+
+  .nav-menu.active {
+    left: 0;
+    margin-top: 20px;
+    background: #3c3a39;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  }
+
+  .nav-menu.active .nav-item {
+    font-weight: 700;
+    line-height: 44px;
+    color: #fff5e1;
+    border-bottom: 1px solid #6f6c6b;
+    padding: 1rem;
+    margin: 0 1.5rem;
+  }
+
+  .icon-mobile.active .bar:nth-of-type(1) {
+    transform: translateY(6.3px) rotate(45deg);
+  }
+
+  .icon-mobile.active .bar:nth-of-type(2) {
+    opacity: 0;
+  }
+
+  .icon-mobile.active .bar:nth-of-type(3) {
+    transform: translateY(-6.3px) rotate(-45deg);
+  }
+}
+
+/* Start styling desktop version */
+@media (min-width: 768px) {
```

Comment:
- [ ] Kindly make sure your page is fully responsive for better accessibility on different screen size as you notice how your page is fucked up on `768px` port width.

### Your design :point_down: 
![image](https://github.com/yesuf023/Portfolio/assets/97100091/da9f4b79-b8eb-4389-8b52-340e23d6a0ef)
![image](https://github.com/yesuf023/Portfolio/assets/97100091/99bb5b6a-37bb-423b-bbba-08a9c99d35ca)
![image](https://github.com/yesuf023/Portfolio/assets/97100091/88ba1c9c-c6a0-404a-9a54-ab6d4507d2f1)
![image](https://github.com/yesuf023/Portfolio/assets/97100091/868b9d99-9f6c-42cb-b2b8-0295cba4bb0d)